internal/core/client/resolver.go

```diff
@@ -43,8 +44,8 @@ func (client *Client) ResolvePath(ctx context.Context, raw string) (*types.Resol
 	}
 
 	res.ReleaseTag = parts[2]
-	res.Release, err = client.GetRelease(ctx, orgID, parts[1], res.ReleaseTag)
-	if err != nil {
+	res.Release, err = client.GetRelease(ctx, orgID, res.RepoName, res.ReleaseTag)
+	if err != nil && !errors.Is(err, types.ErrReleaseNotExist) {
```

Comment:
Similar question to above, what if an error is returned from `GetRelease`, other than `ErrReleaseNotExist`?